<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Marker By Sandeep Manandhar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadImgBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xVal">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yVal">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="width">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="length">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fileNo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No Images</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbuout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUsage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbuout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
